docs/release.md

```diff
@@ -1,31 +1,86 @@
+       To do this, copy the equivalent map entries for the release from the Kubernetes repository.
+       For an example, see the entries [from the v1.15.0 release](https://github.com/kubernetes/kubernetes/blob/v1.15.0/test/utils/image/manifest.go#L202-L252).
+       Within the new function, remove any entries in the `config` map and replace with those copied from the Kubernetes repository.
+       The entries from the Kubernetes repository use an `int` as the key in the map however in the Sonobuoy repository the keys are strings.
+       Convert the new key names to strings.
+     1. To make use of these new images, update the `GetImageConfigs` function within `pkg/image/manifest.go`.
+       Add a new case to the minor version check which will be the minor version of the new Kubernetes release.
+       In this new case, call the newly created function (e.g. `r.v1_15()`).
+   * Update the minimum and maximum Kubernetes API versions that Sonobuoy supports.
+     Edit `pkg/buildinfo/version.go` and update the `MinimumKubeVersion` to be 2 minor version below the new Kubernetes release version and update the `MaximumKubeVersion` to support future point releases.
+     For example, for the Kubernetes 1.15.0 release, the `MinimumKubeVersion` would become `1.13.0` and the `MaximumKubeVersion` would become `1.15.99`.
+1. Commit and open/merge a pull request with these changes.
+1. Create an annotated tag for the commit once the changes are merged:
+    ```
+    git tag -a v0.x.y -m "Release v0.x.y"
+    ```
+1. Push the tag to the [`github.com/heptio/sonobuoy`](https://github.com/heptio/sonobuoy/) repository.
+   * To ensure that the tag is pushed to the correct repository, check which remote corresponds to that repository using the following command:
+     ```
+     git remote -v
+     ```
+     The output of this command should include at least two configured remotes, typically `origin`, which refers to your personal fork, and `upstream` which refers to the upstream Sonobuoy repository.
+     For example:
+     ```
+     origin	git@github.com:<username>/sonobuoy.git (fetch)
+     origin	git@github.com:<username>/sonobuoy.git (push)
+     upstream	https://github.com/heptio/sonobuoy (fetch)
+     upstream	https://github.com/heptio/sonobuoy (push)
+     ```
+     For the following steps, use the remote configured for the `heptio/sonobuoy` repository.
+     The following instructions will use `upstream`.
+    * Push the tag with `git push upstream --tags` (note this will push all tags).
+      To push just one tag, use the following command format (replacing `v0.x.y` with the tag created in the previous step):
+      ```
+      git push upstream refs/tags/v0.x.y
+      ```
+      If there is a problem and you need to remove the tag, run the following commands:
+      ```
+      git tag -d v0.x.y
+      git push upstream refs/tags/v0.x.y
```

Comment:
Didnt we find that we needed to add a colon at the start, since the tag was non-existant?
```
git push upstream :refs/tags/v0.x.y
```

Docs for the push/ref spec is here: https://git-scm.com/docs/git-push but I found https://stackoverflow.com/questions/7303687/why-git-use-the-colon-branch-to-delete-remote-branch to be good at explaining it too. The colon isn't a prefix, it is just helping git understand that there _is no source_ and you're still pushing that non-existent one onto the destination (e.g. delete it)

Reply:
We did! I had forgotten about that so thanks for the reminder! I've included a note with the links you provided in case someone is confused about the ref format in the future.